whirlpool/washerdryer.py

```diff
@@ -107,3 +107,6 @@ def get_cycle_status_rinsing(self):
 
     def get_cycle_status_spinning(self):
         return self.attr_value_to_bool(self.get_attribute(ATTR_CYCLE_STATUS_SPINNING))
+
+    def get_all_attributes(self):
+        return self._data_dict.get("attributes")
```

Comment:
does not seem related. oops? :)

Reply:
Yes, I'm not sure how that got in here, had to do with some testing quite a bit ago. Sorry about that.